scripts/sommelier

```diff
@@ -217,6 +217,18 @@ start_app() {
     fi
     "${WINELOADER}" explorer /desktop="virtual","${native_res}" "${app}" "${ACTION_ARGS[@]}"
     local exit_code="$?"
+  elif [[ -f "${ACTION_ARGS}" ]]; then
```

Comment:
Note: Not entirely sure what the result is of `-f` on an array? Does it return "true" if any argument in the array is a file? Regardless, this "elif" would be gone if you move the for loop to line 202.

Reply:
didn't test that.

Reply:
yeah it skips `elif`.